source/text/layout.hpp

```diff
@@ -29,12 +31,22 @@ class Layout {
 	Layout(int width, Alignment alignment) noexcept;
 	Layout(int width, Truncate truncateType) noexcept;
 	Layout(int width, Alignment alignment, Truncate truncateType) noexcept;
-
+	bool operator==(const Layout &rhs) const noexcept;
+	
+public:
+	static constexpr uint_fast8_t DEFAULT_LINE_HEIGHT = 255;
```

Comment:
255? Or 20?

Reply:
This means the orders to use the default value of the `DrawingSettings`, so it's 255.